### PR TITLE
Implement message filters tag - Closes #1664

### LINKED ISSUE #1664: Implement filter dropdown for filtering transactions by message
This task is part of https://github.com/LiskHQ/lisk-hub/issues/1624 based on designs done in https://github.com/liskhq/lisk-hub/issues/1632

It includes:
  - Dropdown
  - API update
  - Empty results state
  - Filtered results
  - Max length of message error message

### LINKED ISSUE #1667: Implement transaction filter tags
This task is part of #1624 based on designs done in #1633 

  - clicking any tag should do a new API request

### PR DESCRIPTION
**Pull request by michaeltomasik**
### What issue have I solved?

#1664 

### How have I implemented/fixed it?

Implemented Filters

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

File: src/components/transactionsV2/transactionsV2.css [hunk 1]
```diff
@@ -101,3 +105,23 @@
     color: #00b262;
   }
 }
+
+.container {
+  display: flex;
+  background: #f8f9fb;
```
Comment (renatomassao) [inline src/components/transactionsV2/transactionsV2.css]: Needs to add a `border-bottom: 1px solid var(--color-light-gray);`, and remove the one on the: https://github.com/LiskHQ/lisk-hub/blob/de8faa9985745b5f50dcedc292987a909e786ff1/src/components/transactionsV2/transactionsV2.css#L19

File: src/components/transactionsV2/filterButton.css [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  height: 25px;
+}
+
+.error {
+  color: var(--color-error);
+  font-size: var(--button-font-size-s);
+  font-weight: var(--font-weight-semi-bold);
+  text-align: right;
+  width: 100%;
+  text-align: right;
```
Comment (renatomassao) [inline src/components/transactionsV2/filterButton.css]: Remove duplicated
```suggestion
```

File: src/components/transactionsV2/filterButton.css [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  position: absolute;
+  top: 40px;
+}
+
+.errored {
+  border-color: var(--color-error) !important;
+  margin-bottom: 20px !important;
+}
+
+.errored > .inputElement, .errored > .inputElement:focus {
```
Comment (renatomassao) [inline src/components/transactionsV2/filterButton.css]:
```suggestion
.errored > .inputElement,
.errored > .inputElement:focus {
```

File: src/components/transactionsV2/filterButton.css [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  padding: 1px;
+  height: 36px;
+  background-color: var(--color-white);
+  border: 0.5px solid #a6adbd;
+  border-radius: 4px;
+  width: 100%;
+
+  & input {
+    background-color: var(--color-white);
+    line-height: var(--search-box-line-height);
```
Comment (renatomassao) [inline src/components/transactionsV2/filterButton.css]: This variable seems to have no value assigned.

**Comment by Efefefef** [comment]
🐛 Why the data wrapped in % ?
image

@slaweet What about case sensitivity? It is case sensitive now

**Comment by Efefefef** [comment]
👍

**Pull request merged**